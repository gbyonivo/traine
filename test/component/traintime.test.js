import React from 'react';
import { create } from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import { createStore } from 'redux';
import { Provider } from 'react-redux';
import TrainTimes from '../../src/containers/traintimes';
import combinedReducer from '../../src/reducers';
import { stateWithData } from '../__testData__';

describe('TrainTimes', () => {
  it('should render correctly', () => {
    const store = createStore(combinedReducer(), { dataReducer: stateWithData });
    const actual = create(
      <MemoryRouter initialEntries={[{ pathname: '/', key: 'testKey' }]}>
        <Provider store={store}>
          <TrainTimes />
        </Provider>
      </MemoryRouter>
    );
    expect(actual).toMatchSnapshot();
  });
});
