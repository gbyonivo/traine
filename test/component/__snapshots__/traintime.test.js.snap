// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrainTimes should render correctly 1`] = `
<div
  className="traintimes"
>
  <div
    className="loading"
  />
</div>
`;
